docs/cookbook.rst

```diff
@@ -26,27 +26,6 @@ You can override the `path` option by adding the following to your project's cum
             options:
                 path: some_other_dir
 
-Swap out an Ant target for a task
```

Comment:
Not in this PR if you don't want to, but I'd like to replace this with a cookbook entry on using Command task